service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -92,13 +92,14 @@ private void logCacheGetDebugMessages(final PolicyEvaluationRequestCacheKey key,
             final List<String> keys, final List<String> values) {
         if (LOGGER.isDebugEnabled()) {
             LinkedHashSet<String> policySetIds = key.getPolicySetIds();
-            policySetIds.forEach(policySetId -> LOGGER
-                    .debug(String.format("Getting timestamp for policy set: '%s', key: '%s', timestamp:'%s'.",
-                            policySetId, keys.get(0), values.get(0))));
+            policySetIds.forEach(policySetId -> LOGGER.debug(String
```

Comment:
Since we're removing `String.format(..)` in other places, shouldn't we be consistent and remove them from this file as well?